contracts/lib/Structs.sol

```diff
@@ -47,8 +47,9 @@ library Structs {
         bool repeat;
         uint8 id;
         uint8 assetType;
-        address creator;
+        int8 power;
         uint32 epoch;
+        address creator;
```

Comment:
same

Reply:
yes we can do so
but it won't change anything, in both cases storage req remains the same 
let me know if we should change this 

``` 
4 slots - master
    struct job1 { 
        bool active; 1
        bool repeat; 1
        uint8 id; 1
        uint8 assetType; 1
        address creator; 20
        uint32 epoch; 4
        string name; 1
        string selector; 1
        string URL; 1
    } 
    
4 slots - this branch
    struct job2 {
        bool active; 1
        bool repeat; 1
        uint8 id; 1
        uint8 assetType; 1
        uint32 epoch; 4
        address creator; 20
        string name; 1 
        string selector; 1
        string URL; 1
    }
```